app/controllers/api/v1/upload_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+module Api
+  module V1
+    class UploadController < ApplicationController
+      def create
+        upload = Upload.new(
+          search_engine: params[:search_engine],
+          csv_file: params[:csv_file]
```

Comment:
Prefer to use strong params: https://edgeapi.rubyonrails.org/classes/ActionController/StrongParameters.html

So there should be a private method to permit specific params